bin/problem.py

```diff
@@ -176,6 +176,10 @@ def maybe_copy(x):
         for f in in_paths:
             t = run.Testcase(p, f)
             # Require both in and ans files
+            if needinteraction and not t.in_path.with_suffix('.interaction').is_file():
+                if not t.bad_input:
+                    warn(f'Found input file {str(f)} without a .interaction file. Skipping.')
```

Comment:
- I don't think `str(f)` is needed? just `f` should work?
- maybe assert that `needinteraction` is only set if `only_sample` is `True`, and then drop the `if not t.bad_input`?